### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by DaStanzel**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @frenkiboi20

### CODE REVIEW

File: src/com/codersbay/Main.java [hunk 1]
```diff
@@ -4,6 +4,31 @@
 
     public static void main(String[] args) {
 
+        int[] unsorted = {6, 20, 8, 19, 56, 23, 87, 41, 49, 53};
+
+        unsorted = insertionSort(unsorted);
+
+        for (int i = 0; i < unsorted.length; i++) {
```
Comment (gerhofer) [inline src/com/codersbay/Main.java]: Hier gibt's schon eine methode die uns schön den inhalt eines arrays in einen string verwandelt - die [Arrays.toString](https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/util/Arrays.html#toString(int%5B%5D)) Methode

File: src/com/codersbay/Main.java [hunk 2]
```diff
@@ -4,6 +4,31 @@
...
 
+    public static int[] insertionSort(int[] zahlen) {
+
+        int[] sorted = new int[zahlen.length];
+
+        for (int i = 0; i < zahlen.length; i++) {
+            int min = Integer.MAX_VALUE;
+            for (int j = 0; j < zahlen.length; j++) {
+                if (min > zahlen[j]) {
+                    int k = min;
```
Comment (gerhofer) [inline src/com/codersbay/Main.java]: k ist für mich ein `temp` oder `helper` oder so 😉

**Review by gerhofer — CHANGES_REQUESTED** [review]
Du hast hier keinen Insertion Sort implementiert sondern einen Selection Sort 🤓

File: src/com/codersbay/Main.java (hunk 1 above)
Comment (frenkiboi20) [reply src/com/codersbay/Main.java]: Oh sorry, da hab ich mich wohl verguckt : )

File: src/com/codersbay/Main.java [hunk 3]
```diff
@@ -8,27 +10,26 @@ public static void main(String[] args) {
...
 
-        for (int i = 0; i < unsorted.length; i++) {
-            System.out.print(unsorted[i] + ", ");
-        }
+        System.out.println(Arrays.toString(unsorted));
 
     }
 
-    public static int[] insertionSort(int[] zahlen) {
+    public static int[] insertionSort(int array[]) {
```
Comment (gerhofer) [inline src/com/codersbay/Main.java]: Java syntax für arrays ist `int[] array`

File: src/com/codersbay/Main.java [hunk 4]
```diff
@@ -8,27 +10,26 @@ public static void main(String[] args) {
...
+
+        for (int j = 1; j < array.length; j++) {
 
-        int[] sorted = new int[zahlen.length];
+            int key = array[j];
```
Comment (gerhofer) [inline src/com/codersbay/Main.java]: ganz gut, für mich wär der code noch etwas lesbarer wenn wir etwas sprechendere VariablenNamen verwenden würden ;) 
statt `j` würde ich `startOfUnsorted` gut finden, statt `key` `nextUnsortedElement` und statt `i` vlt `greatestElementIndex`

**Review by gerhofer — CHANGES_REQUESTED** [review]

File: src/com/codersbay/Main.java (hunk 4 above)
Comment (frenkiboi20) [reply src/com/codersbay/Main.java]: ok, danke. du hast recht. lässt sich etwas "leichter" lesen. habs geändert..

**Review by gerhofer — APPROVED** [review]
👏

**Pull request closed without merging**